cpu/stm32f0/include/vendor/stm32f051x8.h

```diff
@@ -3278,6 +3278,7 @@ typedef struct
 #define  USART_CR1_DEAT_4                    ((uint32_t)0x02000000)            /*!< Bit 4 */
 #define  USART_CR1_RTOIE                     ((uint32_t)0x04000000)            /*!< Receive Time Out interrupt enable */
 #define  USART_CR1_EOBIE                     ((uint32_t)0x08000000)            /*!< End of Block interrupt enable */
+#define  USART_CR1_M1                        ((uint32_t)0x10000000)            /*!< Word length - Bit 1 */
```

Comment:
Same here about the fact that 7 bit more might just not be supported by this platform, there is no definition in my CubeMX CMSIS file.